### PR TITLE
Show page refactor

### PR DESCRIPTION
**Pull request by jhgould**
## Description
Refactored some of the artist's show pages to add a favorite button. 
Resolves #
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] Refactor (non-breaking change which maintains existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
## Notes
Also added a horizontal scroll bar to the albums section. Need to refactor that so it's horizontal. 
## RSpec Results
```
Finished in 1.18 seconds (files took 3.51 seconds to load)
25 examples, 0 failures
```

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -35,4 +35,10 @@
...
    font-family: "Marker Felt", fantasy;
- }
\ No newline at end of file
+ }
+
+ .scroll_bar{height:1400px;
+             overflow:auto;}
+
+
+
```
Comment (jhgould) [inline app/assets/stylesheets/application.css:44]: Added some css to make a scroll bar. This needs to be a horizontal scroll bar in the future.

File: app/views/artists/show.html.erb [hunk 2]
```diff
@@ -13,6 +14,10 @@
     <%end%>
 </div>
 
+<% if !current_user.favorited?(current_user.id, @artist.id) %>
+  <h3 class="fav-link"> <%= link_to "FAV", "/favorites/#{@artist.id}/new" %>  </h3>  </br>
+<% end %>
+
```
Comment (jhgould) [inline app/views/artists/show.html.erb:20]: reused code to have a favorite link in the show page. The link only appears if the artist is not already in the user's favorites.

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (Gallup93) [reply app/assets/stylesheets/application.css:44]: Good work on the scrollbar! I'm excited to figure out how this front-end world works haha

File: spec/features/favorites/favorites_delete_spec.rb [hunk 3]
```diff
@@ -32,7 +32,7 @@
       expect(page).to have_content(@artist3.name)
 
       within ".artist-#{@artist1.id}" do
-        click_link "remove favorite"
+        click_on "Remove Favorite"
```
Comment (cgaddis36) [inline spec/features/favorites/favorites_delete_spec.rb:35]: nice eye here

File: spec/features/favorites/favorites_new_spec.rb [hunk 4]
```diff
@@ -50,5 +50,32 @@
...
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user1)
+
+      visit "/artists" 
+      within ".artist-header" do 
+        click_on @artist1.name
+      end 
+      click_on "FAV"
+      visit "/favorites"
+      expect(page).to have_content(@artist1.name)
+    end
```
Comment (cgaddis36) [inline spec/features/favorites/favorites_new_spec.rb:79]: nice testing techniques here

**Pull request merged**